packages/fiber/package.json

```diff
@@ -29,11 +29,12 @@
   "main": "dist/react-three-fiber.cjs.js",
   "module": "dist/react-three-fiber.esm.js",
   "types": "dist/react-three-fiber.cjs.d.ts",
-  "react-native": "dist/native.js",
+  "react-native": "native/dist/react-three-fiber-native.cjs",
```

Comment:
I'm able to import from `@react-three/fiber` with this, but I don't receive the typing information unless I import from `@react-three/fiber/dist` which points to its own types.

Is there any way around that or should we encourage people to use `@react-three/fiber/native`?